lib/features/composer/presentation/model/saved_email_draft.dart

```diff
@@ -28,9 +28,10 @@ class SavedEmailDraft with EquatableMixin {
   List<Object?> get props => [
     content,
     subject,
-    toRecipients,
-    ccRecipients,
-    bccRecipients,
+    // Prevent identical Set<EmailAddress>
+    {0: toRecipients},
+    {1: ccRecipients},
+    {2: bccRecipients},
```

Comment:
what is this?

Reply:
Currently `SavedEmailDraft` `props` is a list, in which there are 3 `Set<EmailAddress` `toRecipients`, `ccRecipients` and `bccRecipients`. When dragging any email from one of the `To/CC/BCC` field to another, as long as the whole `props` doesn't change, the hashcodes are the same. By tagging each of those `Set` with an index, this prevent the bug where the code cannot detect changes when an email is dragged across fields.

Reply:
We should write unit test for for it

Reply:
Done